### ISSUE TITLE
Wrong state class

### ISSUE THREAD

**Issue by Tazmanian79** [barleybobs/homeassistant-ecowater-softener#33]
I just have installed a new Ecowater softener and using this integration.
However I receive the following error.

`Entity sensor.ecowater_000_water_available (<class 'custom_components.ecowater_softener.sensor.EcowaterSensor'>) is using state class 'measurement' which is impossible considering device class ('water') it is using; expected None or one of 'total_increasing', 'total'; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.`

Can you check this please?

**Comment by Tazmanian79** [comment]
Any update on this one ?

**Comment by Tazmanian79** [comment]
I have changed sensor.py

Line 54
from state_class=SensorStateClass.MEASUREMENT
to state_class=SensorStateClass.TOTAL

**Comment by olafz** [comment]
Same here

```
Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:496
Integration: Sensor ([documentation](https://www.home-assistant.io/integrations/sensor), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+sensor%22))
First occurred: June 27, 2023 at 22:46:46 (1 occurrences)
Last logged: June 27, 2023 at 22:46:46
Entity sensor.ecowater_water_available (<class 'custom_components.ecowater_softener.sensor.EcowaterSensor'>) is using state class 'measurement' which is impossible considering device class ('water') it is using; expected None or one of 'total_increasing', 'total'; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
```

**Comment by almoney** [comment]
Seems like we are not working this project actively.  We should close and update this code.

**Comment by barleybobs** [comment]
This was fixed in pull request #36.

Thanks 👍
